dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task230420AlterVarcharLengthOfLockedByColTest.java

```diff
@@ -0,0 +1,73 @@
+package com.dotmarketing.startup.runonce;
+
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import org.junit.Test;
+
+import java.sql.SQLException;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class Task230420AlterVarcharLengthOfLockedByColTest {
+    // Change the length to recreate the scenario
+    private void setMinLengthBeforeTask (String tblName) throws SQLException {
+        String query = "";
+        if (DbConnectionFactory.isPostgres()){
+            query = "alter table "+tblName+" alter column locked_by type varchar (36);";
+        }
+
+        final DotConnect dotConnect = new DotConnect();
+        dotConnect.executeStatement(query);
+    }
+
+    // get the modified columns properties
+    private Map<String, String> getColProperties(String tblName) throws DotDataException {
```

Comment:
Why don't you incorporate this into our dotMetadata class ?
is this sql good for both postgres and MS-SQL ?

Reply:
1. The method was added to the metadata class.

![image](https://user-images.githubusercontent.com/127987858/234425745-0e82951c-8aee-4cda-a6f3-d32acc93bbda.png)

2. The query was tested in both SQL versions and works as expected.